### PR TITLE
Updated projects page

### PR DESCRIPTION
**Pull request by bartkalanski**
Signed-off-by: Bart Kalanski <bkalanski@gmail.com>

**Description**

This PR fixes 'Project page update' #1371 

**Notes for Reviewers**

I used CSS grid to get the component look as close to the design as possible, I had a go at using `Row Col Containers`  components to achieve this, but struggled to get it to look anywhere like it should do. 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
There it is!

File: src/sections/Project-grid/index.js [hunk 1]
```diff
@@ -18,62 +18,51 @@ const ProjectPage = ({data}) => {
...
-                            </Link>
-                        </Col>
-                        <Col xs={12} sm={6} lg={4}>
-                            <Link to="/projects/service-mesh-interface-conformance">
-                                <div className="project__block__inner">
-                                    <img src={smi}
-                                        alt="Service Mesh Interface Conformance" 
+                    <div className="project__grid">
+                        {data.allMdx.nodes.map(({id, frontmatter, fields }) => (
+                            <Link to={fields.slug} key={id} className={`project__card ${frontmatter.title === "Service Mesh Performance" ? "five": "six"}`}>
```
Comment (Tanuj22) [inline src/sections/Project-grid/index.js:23]: Can you separate these out like the other @bartkalanski? We are going to create separate pages for them

Comment (bartkalanski) [reply src/sections/Project-grid/index.js:23]: Yes no problem!

**Comment by Tanuj22** [comment]
@bartkalanski some changes have been to the eslint settings. Can you run `npm run lint` and update it?

**Comment by bartkalanski** [comment]
> @bartkalanski some changes have been to the eslint settings. Can you run `npm run lint` and update it?

I've just run it and got 36 warnings regarding unused imports, which are unrelated to the files that I've changed. Is that right ? I'm unsure how to proceed.

**Comment by Tanuj22** [comment]
> > @bartkalanski some changes have been to the eslint settings. Can you run `npm run lint` and update it?
> 
> I've just run it and got 36 warnings regarding unused imports, which are unrelated to the files that I've changed. Is that right ? I'm unsure how to proceed.

You can ignore those and commit the changes

**Comment by bartkalanski** [comment]
@Tanuj22 is that okay ? I had some initial trouble pushing the commit up

**Comment by Tanuj22** [comment]
@bartkalanski can you fix this overflowing icon and we are good to go - 
Screenshot from 2021-01-22 08-32-47

**Review by Tanuj22 — APPROVED** [review]
Thank you @bartkalanski !

**Comment by bartkalanski** [comment]
> Thank you @bartkalanski !

Pleasure!

**Pull request merged**